<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Payments</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Payments for treatment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Cash</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>cash_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="cash_pushButton" >
        <property name="maximumSize" >
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Cheque</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>cheque_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="cheque_pushButton" >
        <property name="maximumSize" >
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Debit Card</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>debitCard_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="debit_pushButton" >
        <property name="maximumSize" >
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Credit Card</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>creditCard_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QPushButton" name="credit_pushButton" >
        <property name="maximumSize" >
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="cash_lineEdit" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="cheque_lineEdit" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="debitCard_lineEdit" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="creditCard_lineEdit" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Other Payments</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Sundries</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>sundries_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="sundries_lineEdit" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Annual HDP</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>annualHDP_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="annualHDP_lineEdit" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Miscellaneous</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>misc_lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="misc_lineEdit" >
        <property name="minimumSize" >
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength" >
         <number>10</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Amount which will appear on receipt  (read only)</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="buddy" >
        <cstring>total_doubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="total_doubleSpinBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="maximum" >
        <double>20000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cash_lineEdit</tabstop>
  <tabstop>cheque_lineEdit</tabstop>
  <tabstop>debitCard_lineEdit</tabstop>
  <tabstop>creditCard_lineEdit</tabstop>
  <tabstop>sundries_lineEdit</tabstop>
  <tabstop>annualHDP_lineEdit</tabstop>
  <tabstop>misc_lineEdit</tabstop>
  <tabstop>total_doubleSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>cheque_pushButton</tabstop>
  <tabstop>credit_pushButton</tabstop>
  <tabstop>debit_pushButton</tabstop>
  <tabstop>cash_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>112</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel" >
     <x>112</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
